e2e/__tests__/nativeEsm.test.ts

```diff
@@ -24,27 +24,35 @@ test('test config is without transform', () => {
 });
 
 test('runs test with native ESM', () => {
-  const {exitCode, stderr, stdout} = runJest(DIR, ['native-esm.test.js'], {
-    nodeOptions: '--experimental-vm-modules --no-warnings',
-  });
+  const {exitCode, stderr, stdout, failed, killed, isCanceled, timedOut} =
+    runJest(DIR, ['native-esm.test.js'], {
+      nodeOptions: '--experimental-vm-modules --no-warnings',
+    });
 
-  const {summary} = extractSummary(stderr);
+  expect(failed).toBe(false);
```

Comment:
Do an object and `toEqual`? That way all props can be inspected instead of bailing at the first one

Reply:
Yep. Just looked at the result and thought to do exactly this (;